roxctl/common/flags/endpoint_test.go

```diff
@@ -0,0 +1,85 @@
+package flags
+
+import (
+	"testing"
+
+	"github.com/stackrox/rox/pkg/env"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/require"
+)
+
+func TestEndpointAndPlaintextSetting(t *testing.T) {
+	truth := false
+	endpointChanged = &truth
+	plaintextSet = &truth
+
+	testCases := []struct {
+		endpoint     string
+		host         string
+		usePlaintext bool
+		err          string
+	}{
+		{
+			endpoint: "localhost:8443",
+			host:     "localhost:8443",
+		},
+		{
+			endpoint: "https://localhost",
+			host:     "localhost:443",
+		},
+		{
+			endpoint: "https://example.com:443",
+			host:     "example.com:443",
+		},
+		{
+			endpoint:     "http://example.com:80",
+			host:         "example.com:80",
+			usePlaintext: true,
+		},
+		{
+			endpoint: "https://example.com",
+			host:     "example.com:443",
+		},
+		{
+			endpoint:     "http://example.com",
+			host:         "example.com:80",
+			usePlaintext: true,
+		},
+		{
+			endpoint:     "http://128.66.0.1",
+			host:         "128.66.0.1:80",
+			usePlaintext: true,
+		},
+		{
+			endpoint: "128.66.0.1",
+			err:      "invalid endpoint: address 128.66.0.1: missing port in address, the scheme should be: http(s)://<endpoint>:<port>",
```

Comment:
```suggestion
			err:      "invalid endpoint: address 128.66.0.1: missing port in address",
```
First, "scheme" is only the "https://" prefix, not the _entire thing_. Second, technically, your suggestion `http(s)://<endpoint>:<port>` is vague because, for example, the `http(s)://` part is optional while `<port>` is not, which is not communicated. I suggest just drop the suggestion for clarity.